.storybook/preview-head.html

```diff
@@ -2,6 +2,10 @@
   document.title = 'Canvas Kit Storybook';
 </script>
 <style type="text/css">
+  html {
+    font-size: 62.5%;
```

Comment:
For Sean:

I liked the idea of base 10 (particularly when we were thinking about how the rems would apply to spacing). However, adding this may be more trouble than it's worth. It may not be as big of an issue within product (where pretty much everything has a px font size specified at some level of the css scope tree. However, within our storybook it caused a lot of changes because we have many headers that do not have an explicit font size applied (meaning they use the browser default of rems, but the rem default is not updated to account for this lower percentage).

If we want to keep base 10, in order to not introduce a bunch of changes to storybook we can either remove this (making anything within the CK storybook that uses the test hierarchy 37.5% larger), or add explicit px or rem sizing for all unstyled text in our stories.

Reply:
Thanks Alex, would it be cleaner for us to use 16/1rem as our base so we wouldn't need to introduce a percentage for the font size which would introduce that added complexity?

Reply:
I'm personally not a big fan of the `62.5%` trick. The default font-size is `16px`, so `16 * .625 = 10` which sets the default font size to `10px`. The percent is used so that if the user changes their base font size to something other than `16px`, everything will automatically scale. It is a convenience to developers (we can think "the design specifies 16px, so it is easy to convert as 1.6rem").

@anicholls did you try adding `font-size: 1.6rem` to the `body` element below?

The reason I'm not a fan of the the `62.5%` trick is the buy-in required by everyone. If you control everything about an application, including the root template and all CSS, it can work well. In our case we do not control everything. It requires every application to set the `font-size` of the `html` and/or `body` elements to these values. I think it makes adoption more difficult.

I think we can use Emotion and SCSS to our advantage and simply make functions. We can do `rem(16)` for a `16px` equivalent size. Should we care what the output value is if we never really have to see it? The input is `16` and the output will be `1rem`. If we did `rem(1)`, the output would be `0.0625rem` (probably for a 1px border). This gives us the benefit of thinking in terms of pixels but using rems. The same advantage of the 62.5% trick, but with no math on our part and without the downside of explicit by-in from everyone.

Reply:
Yeah I think the big argument for the `62.5%` trick is you see readable values when inspecting. But it seems to cause more problems than it's worth. I will update!